src/coprocessor/dag/executor/table_scan.rs

```diff
@@ -330,7 +191,7 @@ mod tests {
         let (snapshot, start_ts) = wrapper.store.get_snapshot();
         let store = SnapshotStore::new(snapshot, start_ts, IsolationLevel::SI, true);
         let mut table_scanner =
-            TableScanExecutor::new(wrapper.table_scan, wrapper.ranges, store, false).unwrap();
+            TableScanExecutor::table_scan(wrapper.table_scan, wrapper.ranges, store, true).unwrap();
```

Comment:
I prefer to use `TableScanExecutor::new` to create a `TableScanExecutor`

Reply:
`TableScanExecutor ` and `IndexScanExecutor` is a type alias for `ScanExecutor`,  and the `new` method is used for create either of these, so I add `table_scan` and `index_scan`, which using `new` internally